### PR TITLE
Standalone wear 2

### PR DESCRIPTION
**Pull request by kskandispersonal**
This is based on standalone-wear:
git checkout -b standalone-wear-2 standalone-wear
git pull upstream master --rebase
git push origin standalone-wear-2

It is better in that it at least does not revert the upstream changes.  Hwr, it is still missing some of the latest upstream code, for example, calibration is missing Note code and NoteSearch.java is missing.  I'm not sure why but maybe they are few enough that I can do the merge manually in AS.

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/CalibrationOverride.java [hunk 1]
```diff
@@ -51,7 +52,7 @@ public void onClick(View v) {
                     if (!TextUtils.isEmpty(string_value)){
                         final double calValue = JoH.tolerantParseDouble(string_value);
 
-                        Calibration last_calibration = Calibration.lastValid();
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/CalibrationOverride.java]: Can you update this to use the new `Calibration.lastValid()`

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java [hunk 2]
```diff
@@ -578,7 +578,6 @@ private void processAndApproveTreatment() {
             updateCurrentBgInfo("approve button");
         }
         processCalibrationNoUI(myglucosenumber, mytimeoffset);
-        staticRefreshBGCharts();
```
Comment (jamorham) [inline app/src/main/java/com/eveningoutpost/dexdrip/Home.java]: Also to paste in this line to your `Home.java`

File: app/src/main/java/com/eveningoutpost/dexdrip/CalibrationOverride.java (hunk 1 above)
Comment (kskandispersonal) [inline app/src/main/java/com/eveningoutpost/dexdrip/CalibrationOverride.java]: I made this change, as well as all of the other missing upstream code.  I think the only diffs now between upstream server code and standalone-wear-2 are my changes.  I hope, that is!

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java (hunk 2 above)
Comment (kskandispersonal) [inline app/src/main/java/com/eveningoutpost/dexdrip/Home.java]: I made this change as well:)

**Comment by jamorham** [comment]
I am working on manually merging this right now. Should I use "kskandipersonal" when referencing you in the commit history or another name?

**Comment by kskandispersonal** [comment]
Yes, you can use "kskandispersonal" or just "kskandis":)

**Comment by jamorham** [comment]
I've been manually merging this and have pushed the changes so far to my master branch, can you resync again and resubmit the PR so we can see how much still remains to do. For some reason it doesn't seem to update when I try to do the comparison.

**Comment by kskandispersonal** [comment]
Will do now...

**Comment by kskandispersonal** [comment]
Just resynced everything.  Only a few changes left to integrate:)

**Comment by jamorham** [comment]
What I don't understand is that I am still seeing 90+ files needing to be changed on the PR and comparing yet I know most of those changes have already been implemented. Do you see the same? Could you try pushing to another branch or something to see if that updates it.

**Comment by kskandispersonal** [comment]
Yes, I see 90 Files Changed, too.  But if I do a compare in AS, there are only a few files different.  Okay, I'll create a new branch a create a new PR on that...

**Comment by kskandispersonal** [comment]
I created a new branch based on this one but it still shows Files Changed:90.  I am trying to do a git rebase upstream but am now finding conflicts.  Let me work through this a bit a see if I can figure out what git is doing....:(

**Comment by jamorham** [comment]
Superseded by https://github.com/jamorham/xDrip-plus/pull/14

**Pull request closed without merging**